### ISSUE TITLE
dpm install errors on Windows x64

### ISSUE THREAD

**Issue by jaysonthepirate** [vutran/dext-core-utils#32]
```
$ dpm install dext-emoji-plugin
events.js:160
      throw er; // Unhandled 'error' event
      ^

Error: spawn npm ENOENT
    at exports._errnoException (util.js:1036:11)
    at Process.ChildProcess._handle.onexit (internal/child_process.js:193:32)
    at onErrorNT (internal/child_process.js:359:16)
    at _combinedTickCallback (internal/process/next_tick.js:74:11)
    at process._tickCallback (internal/process/next_tick.js:98:9)
```

Related to this issue
https://github.com/nodejs/node-v0.x-archive/issues/5841

Possibly fixed by implementing this
https://github.com/IndigoUnited/node-cross-spawn

**Comment by adnasa** [comment]
Which version of __node__ are you using?
because we require that you use __node v6__ for starters

**Comment by jaysonthepirate** [comment]
v6.7.0

**Comment by vutran** [comment]
This should be patched now in `dext-cli@0.3.0`.
